src/components/preferences/Shortcuts.js

```diff
@@ -0,0 +1,118 @@
+/**
+ *
+ * @author Jack Mitt, sriram
+ * Keyboard shortcut user preferences
+ *
+ *
+ */
+import React from "react";
+
+import { injectIntl } from "react-intl";
+import { Field } from "formik";
+
+import ids from "./ids";
+import styles from "./styles";
+import messages from "./messages";
+import prefConstants from "./constants";
+import GridLabelValue from "../utils/GridLabelValue";
+
+import { build, getMessage, FormTextField, withI18N } from "@cyverse-de/ui-lib";
+
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(styles);
+
+function Shortcuts(props) {
+    const { baseId } = props;
+    const classes = useStyles();
+    return (
+        <>
+            <Typography variant="h6" className={classes.sectionHeader}>
+                {getMessage("keyboardShortcutHeaderLbl")}
+            </Typography>
+            <Grid container spacing={2} className={classes.grid}>
+                <GridLabelValue
+                    label={getMessage("appsViewShortcutLbl")}
```

Comment:
am not sure if understand your suggestion here. Can you please elaborate ?

Reply:
Originally I was thinking of using a `label={getMessage(...)}` prop on the on the `FormTextField` instead of using a `GridLabelValue`, but the layout isn't the same.

I was also trying to point out the benefit of using the `label` prop on MUI text fields: if you inspect elements of a MUI text field with this prop set, then you will see that they use a `label` element with a `for` attribute, which should let assistive tech know that's the label for the input field.

This is also what I was getting at with my example in https://github.com/cyverse-de/sonora/pull/122#discussion_r445277180

I thought there were other aria attributes involved, but I might have been thinking of this example instead: https://material-ui.com/api/form-control/